packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantDishes/RestaurantDish.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+    <div :class="[$style['c-restaurantCard-dish']]">
+        <p :class="[$style['c-restaurantCard-dish-description']]">
+            <span
+                data-test-id="restaurant-dish-name"
+                :class="[$style['c-restaurantCard-dish-name']]">
+                {{ name }}
+            </span>
+            <span
+                v-if="calories"
+                data-test-id="restaurant-dish-calories"
+                :class="[$style['c-restaurantCard-dish-calories']]">
+                {{ calories }}
+            </span>
+        </p>
+        <p
+            v-if="price"
+            data-test-id="restaurant-dish-price"
+            :class="[$style['c-restaurantCard-dish-price']]">
+            {{ price }}
+        </p>
+    </div>
+</template>
+
+<script>
+export default {
+    name: 'RestaurantDish',
+    props: {
+        /**
+         * Name of the dish
+         */
```

Comment:
do we need this comment? the following two comments have some useful information about the format, but I wonder if we can also avoid them somehow too.